dvc/tree/local.py

```diff
@@ -242,6 +243,8 @@ def get_file_hash(self, path_info):
     def getsize(path_info):
         return os.path.getsize(path_info)
 
+    _upload_fobj = partialmethod(copy_fobj, chunk_size=BaseTree.CHUNK_SIZE)
```

Comment:
Btw, we have `copy_fobj` method here and in BaseTree, do we still need it or is it the same as upload_fobj?

Reply:
`copy_fobj` is basically `uplad_fobj` *without* progress bar.

Reply:
we could change all occurences to use `upload_fobj(no_progress_bar=True)`

Reply:
+ and also without chunking

Reply:
@isidentical It not having a progress bar is probably a bug :)